api/src/pcapi/core/object_storage/backends/gcp.py

```diff
@@ -146,6 +146,69 @@ def get_public_object(self, folder: str, object_id: str) -> bytes:
             )
             raise exc
 
+    def object_exists(self, folder: str, object_id: str) -> bool:
+        storage_path = folder + "/" + object_id
+        try:
+            bucket = self.get_gcp_storage_client_bucket()
+            gcp_cloud_blob = bucket.blob(storage_path)
+            return gcp_cloud_blob.exists()
+
+        except Exception as exc:
+            logger.exception(
+                "An error has occurred while trying to check file existence on GCP bucket",
+                extra={
+                    "exc": exc,
+                    "project_id": self.project_id,
+                    "bucket_name": self.bucket_name,
+                    "storage_path": storage_path,
+                },
+            )
+            raise exc
+
+    def copy_object_to(
+        self,
+        source_folder: str,
+        source_object_id: str,
+        destination_backend: "GCPBackend",
+        destination_folder: str,
+        destination_object_id: str,
+    ) -> None:
+        source_path = f"{source_folder}/{source_object_id}"
+        destination_path = f"{destination_folder}/{destination_object_id}"
+        try:
```

Comment:
J'ai l'impression que l'on pourrait simplifier ainsi non ?
```suggestion
    def copy_object_to(
        self,
        source_path: str,
        destination_backend: "GCPBackend",
        destination_path: str,
    ) -> None:
        try:
```